page-search.php

```diff
@@ -35,9 +35,11 @@
 				<?php endif; ?>
 			</div>
 
-			<div class="content-main flex-container">
+			<div id="content" class="content has-sidebar">
 				<?php get_template_part( 'content', 'pagefull' ); ?>
+				<?php get_sidebar(); ?>
 			</div>
 		</div>
 		<?php endwhile; // end of the loop. ?>
```

Comment:
The while was replaced to its proper place and I noticed while studying the entire template that its breadcrumb was wrong and it used code that must have been copied from other templates but served no purpose, so I fixed both.